src/lib/tests/test_kernels.cpp

```diff
@@ -111,3 +111,54 @@ TYPED_TEST(Coordinates, CalcBondsMatchesVanillaInBox) {
     EXPECT_EQ_T(this->results[i], this->ref[i]);
   }
 }
+
```

Comment:
We should aim to cover as many of the dispatch paths as we can bit for now looks great.